src/helpers/sendEmail.js

```diff
@@ -21,3 +21,24 @@ const sendEmail = async (businessEmail, requester, request) => {
   sendGrid.send(msg);
 };
 export default sendEmail;
+
+// sgMail.setApiKey(process.env.BN_API_KEY);
```

Comment:
hey @Kagorora , I think these comments should be deleted!?

Reply:
@T2Wil I have removed it